apps/emails/management/commands/email.py

```
@@ -0,0 +1,427 @@
+
+    def _search_email(self, options: dict[str, Any]) -> None:
+        """Search inbound messages and print results as JSON."""
+
+        limit = options.get("search_limit")
+        try:
+            limit = int(limit) if limit is not None else 10
+        except (TypeError, ValueError):
+            limit = 10
+
+        if limit <= 0:
+            raise CommandError("--search-limit must be a positive integer.")
```

Ruff (0.15.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)